README.md

```diff
@@ -14,357 +14,68 @@ For more information on HMDA, checkout the [About HMDA page](http://www.consumer
-
-```shell
-
-> project cluster
-> clean
-> ~re-start
-```
-
-Confirm that the platform is up and running by browsing to http://localhost:8080
-
-When finished, press enter to get the sbt prompt, then stop the project by entering `reStop`.
-
-
-### Running the Project with Docker
-
-#### To run only the API
 
-First, ensure there's a compiled jar to create the Docker image with:
-```shell
-sbt clean assembly
-```
+### Building and runnint the .jar
 
-Build the docker image
-```shell
-docker build -t hmda-api .
-```
+* To build JVM artifacts (the default, includes all projects), from the sbt prompt:
 
-Then, run the docker image
 ```shell
-docker run -d -p "8080:8080 -p 8082:8082" hmda-api
+> clean assembly
 ```
+This task will create a `fat jar`, which can be executed on any `JDK9` compliant `JVM`:
 
-The Filing API will run on `$(docker-machine ip):8080`
-The Public API will run on `$(docker-machine ip):8082`
-
-By default, the `HDMA Platform` runs with a log level of `INFO`. This can be changed by establishing a different log level in the `HMDA_LOGLEVEL` environment variable.
-For the different logging options, see the [reference.conf](https://github.com/akka/akka/blob/master/akka-actor/src/main/resources/reference.conf#L38) default configuration file for `Akka`.
-
-#### API Load Testing
+`java -jar target/scala-2.12/hmda2.jar`
 
-A load testing scenario for the API has been built using [Gatling](https://gatling.io/). The parameters for this test can be configured in the `cluster/src/test/resources/application.conf` file.
-To run the default scenario:
+### Building and running the Docker image
 
-```shell
-$ sbt
-> project cluster
-> gatling:test
-```
-
-After the test is run, some statistics will be presented on the screen. To further study the results of the test, a report can be opened on a browser from the sbt prompt:
+* To build a `Docker` image that runs as a single node cluster, from the sbt prompt:
 
 ```shell
-> gatling:lastReport
+> docker:publishLocal
 ```
+This task will create a `Docker` image. To run a container with the `HMDA Platform` as a single node cluster, will all dependencies:
 
-** NOTE: The load test requires the sample panel [file](loader/src/main/resources/inst_data_2017_dummy.csv) to be loaded before running it.
-
-#### To run the entire platform
-
-1. Ensure you have a Docker Machine with sufficient resources, as described in the [Docker](#docker) section above.
-
-1. Clone [hmda-platform-ui](https://github.com/cfpb/hmda-platform-ui) and
-    [hmda-platform-auth](https://github.com/cfpb/hmda-platform-auth) into the same
-    directory as hmda-platform.
-
-        ~/dev/hmda-project$ ls -l
-        drwxr-xr-x  22 lortone  staff   748B Jul 27 16:28 hmda-platform/
-        drwxr-xr-x  25 lortone  staff   850B Jul 25 17:13 hmda-platform-ui/
-        drwxr-xr-x  23 lortone  staff   796B Jul 28 17:15 hmda-platform-auth/
+`docker run --rm -ti -p 8080:8080 -p 8081:8081 -p 8082:8082 -p 19999:19999 cfpb/hmda`
```

Comment:
Following the instructions here, I think this line needs to be:
`docker run --rm -ti -p 8080:8080 -p 8081:8081 -p 8082:8082 -p 19999:19999 hmda/hmda-platform`